Makefile.am

```diff
@@ -194,6 +196,41 @@ cc_oci_runtime_CFLAGS = \
 	$(LIBMNL_CFLAGS) \
 	$(UUID_CFLAGS)
 
+libexec_SCRIPTS = cc-proxy
+
+AM_V_GO    = $(am__v_GO_@AM_V@)
+am__v_GO_  = $(am__v_GO_@AM_DEFAULT_V@)
+am__v_GO_0 = @echo "  GOBUILD " $@;
+am__v_GO_1 =
+
+systemdservice_in_files =		\
+	proxy/cc-proxy.service.in	\
+	proxy/cc-proxy.socket.in	\
+	$(NULL)
+
+systemdservice_files = 		\
+	proxy/cc-proxy.service	\
+	proxy/cc-proxy.socket	\
+	$(NULL)
```

Comment:
Oh but that one is a different problem. The symlinks in the  multi-user.target.wants directory control which services are started at boot time. That decision doesn't belong to the "make install" time, but to the admin/user of that machine or to the packaging system (deciding which services should be started by default in a distribution) . Same thing for sockets.target.wants.

The way to control that is through `systemctl enable`. For us that will be

```
$ sudo systemctl enable cc-proxy
Created symlink from /etc/systemd/system/multi-user.target.wants/cc-proxy.service to /usr/lib/systemd/system/cc-proxy.service.
```

and

```
$ sudo systemctl enable cc-proxy.socket
Created symlink from /etc/systemd/system/sockets.target.wants/cc-proxy.socket to /usr/lib/systemd/system/cc-proxy.socket.
```